app/src/main/java/com/inveitix/android/clue/ui/MapActivity.java

```diff
@@ -65,7 +63,7 @@ public void onDoorClicked(Door door) {
                     intent.putExtra(EXTRA_MUSEUM_ID, museumId);
                     intent.putExtra(EXTRA_ROOM_ID, door.getConnectedTo());
                     intent.putExtra(EXTRA_PREVIOUS_ROOM_ID, room.getId());
-                    intent.setFlags(Intent.FLAG_ACTIVITY_NO_HISTORY);
+                    //intent.setFlags(Intent.FLAG_ACTIVITY_NO_HISTORY); bugging when scan no entrance room but other and try to find the qr
```

Comment:
Don't commit commented code.
If this code should stay - uncomment it,
if not -delete it